bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -136,54 +147,51 @@ final class BagsManager(executorService: ExecutorService) {
           // or (b) we don't care about that program anymore, so we're done.
           if (dequeue(key, obs.getProgramID)) {
             // Otherwise construct an obs context, verify that it's bagworthy, and go
-            ObsContext.create(obs).asScalaOpt.filter(ctx =>
-              isEligibleForBags(ctx)
-              && hasBeenUpdated(obs, ctx)
-              && notObserved(obs)
-            ).foreach { ctx =>
-              //   do the lookup
-              //   on success {
-              //      if we're in the queue again, it means something changed while this task was
-              //      running, so discard this result and do nothing,
-              //      otherwise update the model
-              //   }
-              //   on failure enqueue again, maybe with a delay depending on the failure
-              val bagsIdMsg = s"BAGS lookup on thread=${Thread.currentThread.getId} for observation=${obs.getObservationID}"
-              LOG.info(s"Performing $bagsIdMsg.")
+            ObsContext.create(obs).asScalaOpt.foreach { ctx =>
+              if (isEligibleForBags(ctx)
+                && hasBeenUpdated(obs, ctx)
+                && notObserved(obs)) {
```

Comment:
If I read this correctly this means that observed observations will have the auto guide star they used wiped out?  That may not have been intended?

Also, won't this wipe out the auto guide star for otherwise eligible observations that have been updated but not in a way that impacts the AGS selection?